.zuul.yaml

```diff
@@ -23,11 +23,17 @@
       watcher_hook: "{{ watcher_repo }}/ci/playbooks/deploy_watcher_service.yaml"
       watcher_coo_hook: "{{ watcher_repo }}/ci/playbooks/deploy_cluster_observability_operator.yaml"
       run_tempest: false
-      # Based on current testing, https://github.com/openstack-k8s-operators/watcher-operator/pull/47#issuecomment-2607474033
-      # We need decision engine and applier CRD to ready to run
-      # whole test suite
       cifmw_test_operator_tempest_include_list: |
         watcher_tempest_plugin.tests.api.admin.test_api_discovery.TestApiDiscovery
+        watcher_tempest_plugin.*
+      # We need to exclude client_functional tests until we have watcherclient installed in the
+      # tempest container.
+      # Strategies execution tests are failing. Excluding until the work on the watcher-tempest-plugin
+      # is finished upstream.
+      cifmw_test_operator_tempest_exclude_list: |
+        tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_mtu_sized_frames
```

Comment:
As we are not running tempest.scenario tests. I think we can remove `tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_mtu_sized_frames` from skiplist.